### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by nottengame** [anapnoe/stable-diffusion-webui-ux#93]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Hello! 
Thank you for your great UI project! 
I'd wanted to offer to add an ability to set up custom paths to SD models, loras, textual inversions, etc. 
Such functionality is available in Vladomatic and it's very handy.

### Proposed workflow

1. Go to settings -> Stable Diffusion
2. Press the checkbox "set a custom path for SD models"
3. Input or select via a standard dialog window a path for root directories where models are located 
For example: "C:\Stable-Diffusion\vlad1111\automatic\models"

**Comment by Pixl64** [comment]
This is an option is the webui-user.bat 

https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings

**Comment by anapnoe** [comment]
Hello you are welcome
it would also be available to set it up in option settings we are collaborating with @vladmandic 
stay tuned!

**Comment by midcoastal** [comment]
So...  I hope there will be a way to do this at the command-line and over-write whatever is in config...

I launch SD two different ways...  One way with one set of directories (let's say...  SFW), and open to some friends and family...  And another way that is not open to friends and family.  I do this via separate batch files.  Will that still be possible?

**Comment by vladmandic** [comment]
@midcoastal see <https://github.com/vladmandic/automatic/issues/477#issuecomment-1521649245>

**Comment by midcoastal** [comment]
@vladmandic - 'preciate you.

REALLY, the feature(s) I really want, are sessions + some kind of login/users or something (could even be as basic as just one privileged user): user sessions can only view images *they* generated, and a way to hide models (SD, Lora, Lycoris, Embeding, Hyper, etc) from all but the privileged login(s) (which would be able to view all generated images).

Maybe even a way to lock the UI down for non-privileged users (visible tabs/settings, and what can and can not be changed).

That's the bullet, right there.  But a TOTALLY different conversation...  lol

**Comment by vladmandic** [comment]
yeah, i get it - and i'd love it as well, its just there is soooo much legacy code that assumes once you're logged in, there are no more checks. it will get done, but its not an easy request nor a short term one.
